micro-ui/web/micro-ui-internals/packages/react-components/src/atoms/svgindex.js

```
@@ -1837,6 +1837,48 @@ const TreatmentQualityIcon = ({ className = "", fill = "", style = {} }) => (
   </svg>
 );
 
+
+const BarsChart = ({ className = "", fill = "", style = {} }) => (
+  <svg width="20" height="18" className={className} onClick={onClick} style={style} viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path d="M0 18V16H20V18H0ZM1 15V8H4V15H1ZM6 15V3H9V15H6ZM11 15V6H14V15H11ZM16 15V0H19V15H16Z" fill={fill} />
+  </svg>
+
+);
+const Employee = ({ className = "", fill = "", style = {} }) => (
+  <svg width="25" height="26" viewBox="0 0 25 26" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)